src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests_DefaultsSource.vb

```diff
@@ -586,6 +586,41 @@ $$
             End Using
         End Function
 
+        <WpfFact>
+        Public Async Function TestSelectionOfPromotedDefaultItemWithEmptyFilterText() As Task
+            Using state = TestStateFactory.CreateCSharpTestState(
+                <Document>
+using System;
+using System.Text;
+public class Program
+{
+    static string Title { get; } = "";
+    static void Main(string[] args)
+    {
+        var sb = new StringBuilder();
+        sb.Append$$
+    }
+}
+                </Document>,
+                showCompletionInArgumentLists:=True,
+                extraExportedTypes:={GetType(MockDefaultSource)}.ToList())
+
+                MockDefaultSource.Defaults = ImmutableArray.Create("Title")
+                state.SendTypeChars("(")
+
+                Dim expectedItems1 = New List(Of ValueTuple(Of String, Boolean)) From {
+                    CreateExpectedItem("★ Title", isPromoted:=True),
+                    CreateExpectedItem("Title", isPromoted:=False)
+                }
+
+                AssertCompletionItemsInRelativeOrder(state, expectedItems1)
+                Await state.AssertSelectedCompletionItem("★ Title", isHardSelected:=True)
```

Comment:
This is current behavior: we always hard select item suggested by WLC when there's no filter text
@dpugh @jasonmalinowski